src/GraphQL.Conventions/GraphQL.Conventions.csproj

```diff
@@ -27,13 +27,11 @@
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <LangVersion>7.3</LangVersion>
-  </PropertyGroup>
-
   <ItemGroup>
-    <PackageReference Include="GraphQL" Version="3.3.2" />
-    <PackageReference Include="GraphQL.NewtonsoftJson" Version="3.3.2" />
+    <PackageReference Include="GraphQL" Version="4.0.2" />
```

Comment:
Wanna bump the GraphQL.Conventions package version on lines 5 and 6, plus in `CommonAssemblyInfo.cs`? Let's bump major to 4 :-)